src/PixiEditor/Views/Dialogs/HelloTherePopup.xaml

```diff
@@ -188,24 +188,35 @@
                                                         <StackPanel Orientation="Horizontal" VerticalAlignment="Bottom" HorizontalAlignment="Center">
-                                                            <Button Margin="0,0,0,5" Height="25" Width="25"
-                                                                    Command="{Binding DataContext.OpenInExplorerCommand, RelativeSource={RelativeSource AncestorType=uc:AlignableWrapPanel}}"
+                                                            <StackPanel.Resources>
+                                                                <Style TargetType="Button" BasedOn="{StaticResource BaseDarkButton}">
+                                                                    <Setter Property="Margin" Value="0,0,0,5"/>
+                                                                    <Setter Property="Width" Value="25"/>
+                                                                    <Setter Property="Height" Value="25"/>
+                                                                    <Setter Property="MinWidth" Value="25"/>
+                                                                    <Setter Property="MinHeight" Value="25"/>
+                                                                    
+                                                                    <Style.Triggers>
+                                                                        <Trigger Property="IsMouseOver" Value="False">
+                                                                            <Setter Property="Background" Value="Transparent"/>
+                                                                        </Trigger>
+                                                                        <Trigger Property="IsMouseOver" Value="True">
+                                                                            <Setter Property="Background" Value="#70FFFFFF"/>
+                                                                        </Trigger>
+                                                                    </Style.Triggers>
+                                                                </Style>
+                                                            </StackPanel.Resources>
+                                                            <Button Command="{Binding DataContext.OpenInExplorerCommand, RelativeSource={RelativeSource AncestorType=uc:AlignableWrapPanel}}"
                                                                     CommandParameter="{Binding FilePath}"
                                                                     ToolTip="Open in File Explorer">
-                                                                <TextBlock Text="&#xEC50;" FontFamily="Segoe MDL2 Assets"
+                                                                <TextBlock Text="&#xEC50;" FontFamily="{DynamicResource NativeIconFont}"
```

Comment:
![image](https://user-images.githubusercontent.com/25402427/236305311-b4087a0f-f3ef-4a97-9438-27e034f8ca3b.png)

is this supposed to look like this?

Reply:
that's the segoe fluent close icon. I guess it's kinda thin so we might as well use the MDL2 assets for that one